### DISCUSSION TITLE
Current State of this?

### DISCUSSION THREAD

**Discussion by ooJaan** [j0hnwang/OMEN-Transcend-16-ACPI-fix#4] [Q&A]
Hi, thinking of buying this laptop, how well is linux currently supported on it? Have HP fixed their thing or do I still need patch it manually?
Greetings,
Jan

**Comment by jatofi-official** [comment]
I have bought a HP OMEN Transcend 16 recently, and no this fix does not work. Unfortunately I have to boot using noapic, so tracpad and built in speakors do not work.
